scripts/build-backend.sh

```
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# FoodiePass Backend Build Script
+
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)